### PR TITLE
Add functionality to constantly deplete fuel

### LINKED ISSUE #57: Constantly deplete fuel while ship is flying in overworld
- Depends on #56 

Tasks:

- [ ] Constantly deplete fuel while the ship is driving

### PR DESCRIPTION
**Pull request by fredelberger**
Fuel is constantly consumed each fixed update

### Closes

- Closes #57

### CODE REVIEW

File: Assets/Scenes/Pilot/PilotManager.cs [hunk 1]
```diff
@@ -60,4 +60,9 @@ private void EndGame(bool gameEndedSuccessfully)
             SceneManager.LoadScene("GameOver");
         }
     }
+
+     private void FixedUpdate()
```
Comment (mario-nowak) [inline Assets/Scenes/Pilot/PilotManager.cs]: The indentation in this line looks weird. Can you fix that?

File: Assets/Scenes/Pilot/PilotManager.cs [hunk 2]
```diff
@@ -60,4 +60,9 @@ private void EndGame(bool gameEndedSuccessfully)
...
+    {
+        DepleteFuel(0.00007f);
```
Comment (mario-nowak) [inline Assets/Scenes/Pilot/PilotManager.cs]: We should probably extract this number into a variable and then multiply it by `Time.FixedDeltaTime` here. Could you change that?

**Review by mario-nowak — CHANGES_REQUESTED** [review]
Cool!

**Comment by mario-nowak** [comment]
Don't forget to re-request a review once you are done :)

**Review by Gauteee — APPROVED** [review]

**Pull request merged**